lib/internal/Magento/Framework/Mview/Config/Converter.php

```diff
@@ -28,6 +28,8 @@ public function convert($source)
             $data['view_id'] = $viewId;
             $data['action_class'] = $this->getAttributeValue($viewNode, 'class');
             $data['group'] = $this->getAttributeValue($viewNode, 'group');
+            $data['store_scope'] = $this->getAttributeValue($viewNode, 'store_scope');
```

Comment:
Explicit knowledge about the application entities such as a store, an attribute, should not be mentioned at the Framework level. By the way, these are not the only scopes available in Magento. Could you please make it more configurable, to make sure that adding of a new scope, like a shared catalog will not require code changes.

Reply:
"shared catalog"  rely on existing  data. 
We can rename it to "scope" to be more abstract, but I do not see any reason to provide complex configuration which does not have any use-cases